Hunting Queries/Microsoft 365 Defender/Exfiltration/unusual-volume-of-file-sharing.yaml

```diff
@@ -1,10 +1,10 @@
 id: bafc1446-1cc4-4f6d-ad76-1250b8c3b60c
-name: Unusual volume of file deletion by user.
+name: Unusual volume of external file sharing with external user.
 description: |
-  This query looks for users performing file deletion activities. Spikes in file deletion observed from risky sign-in sessions are flagged here.
+  This query looks for users sharing access to files with external users. 
   This applies to SharePoint and OneDrive users.
-  Audit event and Cloud application identifier references.
-  Reference - https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-activities?view=o365-worldwide
+  Audit event and Cloud application identifier references.  
+  Reference - https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-sharing?view=o365-worldwide
```

Comment:
Forgot to inform in the PR.. Please remove  locale en-us from all links..  For some reason, DocumentsLinkValidation test did not flag it.  The URL without locale will work for all customers and redirect to correct locale.

Reply:
Removed the language locale from the references.